Office365APIEditor/UI/AccessTokenWizard.cs

```diff
@@ -624,6 +632,26 @@ private void button_Next_Click(object sender, EventArgs e)
                     
                     break;
 
+                case PageIndex.Page14_SharePointOnlineAppOnlyOptionForm:
+                    // Option form for the SharePoint Online App-Only REST API Microsoft Azure Access Control Service application.
+
+                    if (ValidateSharePointOnlineAppOnlyByKeyParam())
+                    {
+                        TokenResponse tokenResponse = AcquireSharePointOnlineAppOnlyAccessTokenByKey();
+
+                        if (tokenResponse != null)
+                        {
+                            SaveSettings();
+
+                            // Create a return value and close this window.
+                            clientInfo = new ClientInformation(tokenResponse, AuthEndpoints.OAuthV1, Util.ConvertResourceNameToResourceEnum(comboBox_Page08_Resource.SelectedText), textBox_Page08_ClientID.Text, "", "", "");
```

Comment:
The 3rd and 4th parameter of ClientInformation should be changed to the right value. Probably, textBox_Page08_ClientID.Text should be textBox_Page14_ClientID.Text?

Setting Null or whitespace to the 3rd parameter is OK, but we might need to define the appropriate resource in Util.ConvertResourceNameToUri, Util.ConvertResourceNameToResourceEnum and Util.ConvertResourceEnumToResourceName, because I slightly changed the behavior of scintilla_HotspotClick() and it uses the value of ClientInformation in https://github.com/microsoft/Office365APIEditor/commit/df71fa1a5e10791341d5292b8f3909c55bd8db2b .

(We don't need to change Util.ResourceNames because this is used for the source of the dropdown lists of access token wizard.)

Reply:
yes, thanks for the tip, applying fix. As of HotSpot click I may suppose we can leave it as it is as there are not a JSON lexer and the URLs are mostly like <tenantname>.sharepoint.com or schemas.microsoft.com

Reply:
yes, just double checked, the response is XML and URLs in the body are not trapped by HotspotClick so no affect here